metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_query.py

```diff
@@ -105,6 +105,50 @@ def tables_for_schema(schema_name: str, db_name: Optional[str]) -> str:
         and table_type in ('BASE TABLE', 'EXTERNAL TABLE')
         order by table_schema, table_name"""
 
+    @staticmethod
+    def get_all_tags_on_object(
```

Comment:
is this without propagation or with?

I think using the _without_propagation or _with_propagation suffixes will help with clarity